.pre-commit-config.yaml

```
@@ -0,0 +1,52 @@
+---
+repos:
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.275"
+    hooks:
+      - id: ruff
+        args: [--fix, --exit-non-zero-on-fix]
+  - repo: https://github.com/psf/black
+    rev: 23.3.0
+    hooks:
+      - id: black
```

Add the name to make it more readable and beautiful when running
```suggestion
      - id: black
        name: Black (Python formatter)
```